LayoutTests/http/tests/paymentrequest/ApplePayModifier-disbursementPaymentRequest.https.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Tests for providing `disbursementPaymentRequest` as part of `ApplePayModifier`.</title>
+<script src="/js-test-resources/ui-helper.js"></script>
+<script src="/resources/payment-request.js"></script>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+setup({ explicit_done: true, explicit_timeout: true });
+
+test((test) => {
+    const method = validPaymentMethod();
+    method.data.features = ["paymentRequestDisbursements"];
+    new PaymentRequest([method], validPaymentDetails());
+}, "Should have a feature for `disbursementPaymentRequest`.");
+
+let lineItems = {
+    total: {
+        "label": "Chime",
+        "amount": "15.00"
+    },
+    ifo: {
+        "label": "Instant Transfer Fee",
+        "amount": "0.15",
+        "disbursementLineItemType": "instantFundsOutFee"
+    },
+    disbursement: {
+        "label": "Total Amount",
+        "amount": "14.85",
+        "disbursementLineItemType": "disbursement"
+    }
+}
+
+let validLineItems = [ lineItems.total, lineItems.ifo, lineItems.disbursement ]
+
+function modifiersWithData(data, lineItems) {
+    data.additionalLineItems = lineItems
+    return [ { supportedMethods: 'https://apple.com/apple-pay', data } ];
+}
+
+function testValidDisbursementPaymentRequest(description, {initialDisbursementPaymentRequest, expectedDisbursementPaymentRequest}) {
```

Comment:
This is usually a bit more maintainable, plus makes it clear the properties won't be reassigned:
 
```suggestion
function testValidDisbursementPaymentRequest(description, options) {
   const {initialDisbursementPaymentRequest, expectedDisbursementPaymentRequest} = options;
```